src/functionalTest/kotlin/com/github/jengelman/gradle/plugins/shadow/transformers/MergeLicenseResourceTransformerTest.kt

```diff
@@ -0,0 +1,59 @@
+package com.github.jengelman.gradle.plugins.shadow.transformers
+
+import assertk.assertThat
+import assertk.assertions.isEqualTo
+import com.github.jengelman.gradle.plugins.shadow.testkit.containsOnly
+import com.github.jengelman.gradle.plugins.shadow.testkit.getContent
+import kotlin.io.path.appendText
+import kotlin.io.path.writeText
+import kotlin.text.lines
+import org.junit.jupiter.api.Test
+
+class MergeLicenseResourceTransformerTest : BaseTransformerTest() {
+
+  @Test
+  fun twoLicenses() {
```

Comment:
As there is only one test in the class, I'd prefer to move it into 

https://github.com/GradleUp/shadow/blob/a304484b683321c4d6919efdfb175fdd80d7a75a/src/functionalTest/kotlin/com/github/jengelman/gradle/plugins/shadow/transformers/TransformersTest.kt#L26